packages/web3torrent/src/components/channel-list/channel/Channel.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import {text, withKnobs} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+import '../../../App.scss';
+import '../ChannelList.scss';
+import {Channel} from './Channel';
+import './Channel.scss';
+
+storiesOf('Web3Torrent', module)
+  .addDecorator(withKnobs())
+  .add('Channel', () => (
+    <table className="channel-list">
+      <tbody>
+        <Channel
+          channel={{
+            channelId: text('Channel ID', '0x95da8', 'Channel State'),
```

Comment:
What's the purpose of the `'Channel State'` part?

Reply:
This is for the storybook to group the knobs together.

![image](https://user-images.githubusercontent.com/2715151/75595994-9ca87c00-5a5c-11ea-9e58-2078470e0450.png)

If assigning different groups, these knobs won't show up in one tab, like this:

![image](https://user-images.githubusercontent.com/2715151/75596046-daa5a000-5a5c-11ea-9d97-95f146abf198.png)

Reply:
Is it possible to just omit the group label instead of using a single one?

Reply:
You are right. The label is not needed at all. Fixed.